obspy/core/inventory/response.py

```diff
@@ -1551,7 +1551,7 @@ def approximation_type(self, value):
         self._approximation_type = value
 
 
-class FilterCoefficient(CustomFloat):
+class FilterCoefficient(FloatWithUncertainties):
```

Comment:
> Is this correct? I don't think it should have uncertainties.

From looking at the xsd I think it should..

![screenshot from 2016-07-11 11 52 35](https://cloud.githubusercontent.com/assets/1842780/16726758/2baa22d4-475e-11e6-9343-efff82e71c88.png)
![screenshot from 2016-07-11 11 52 15](https://cloud.githubusercontent.com/assets/1842780/16726759/2bca7c3c-475e-11e6-99ad-1a813145e3cc.png)